poetry/console/commands/task.py

```diff
@@ -0,0 +1,20 @@
+from .env_command import EnvCommand
+
+
+class TaskCommand(EnvCommand):
+    """
+    Runs a task in the appropriate environment.
+
+    task
+        { task : The task to run.}
+    """
+
+    def handle(self):
+        task = self.argument("task")
+        tasks = self.poetry.local_config.get("tasks")
+
+        if tasks and task in tasks:
+            return self.run_task(tasks[task])
+
+    def run_task(self, task):
+        return self.env.run(*task.split(), shell=True, call=True)
```

Comment:
Should this be run through [`shlex`](https://docs.python.org/3/library/shlex.html) first? `shlex` had `punctuation` chars added in 3.6, which I can't find a backports for.

Would this work with a command such as `echo 'Hello, ' && echo 'world!'`? i.e. would the space in the first quoted string get separated into `'Hello,` and `'` and would the `&&` work?

[`yarn`](https://github.com/yarnpkg/yarn/blob/7f419103307da7856627c4ca078460234701eb75/src/cli/commands/run.js#L81) uses [`puka`](https://www.npmjs.com/package/puka)

Reply:
Oh, just looked at the test below. Obviously does work with commands like above.